src/plugins/navigation_embeddable/server/saved_objects/navigation_embeddable.ts

```diff
@@ -32,6 +32,7 @@ export const navigationEmbeddableSavedObjectType: SavedObjectsType = {
         dynamic: false,
         properties: {},
       },
+      layout: { type: 'text' },
```

Comment:
You're definitely correct that it should be a `keyword` field - good catch! Updated in 8cfeb29b7daa3eb9a174a73724c34e241ed1cac3

> Are we planning to query/sort by this field? If not, I'd rather not declare it in the mappings (it can be added when needed).

No, we're not planning on being able to query/sort on this field - added `index: false` to the mapping to reflect this. Not entirely sure what you mean by not adding it to the mappings in the first place, though... We need this value saved as part of the saved object, so how do we exclude it from the saved object mapping? I've pinged @nickpeihl since he did a majority of the content management work and has a better idea of how it all works 😅

Reply:
***Update:** Seems like adding `dynamic: false` to the `mappings` field fixes the above error (done 8b4e2403645a96fd9d33a03f59acabb8259fc7b2) - I see that the [docs](https://docs.elastic.dev/kibana-dev-docs/tutorials/saved-objects) has this, and we may have just missed it in the initial content management PR.